spec/features/navigation/navigation_spec.rb

```diff
@@ -1,22 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe 'Site Navigation' do
-  before :each do
 
-    visit '/'
+  it 'shows this information from the nav bar' do
+  visit '/'
+      within('#top-nav') do
```

Comment:
Edit first test to be able to see stuff from nav bar as a regular, not-logged-in, visitor